lib/handle-pull-request-change.js

```diff
@@ -3,29 +3,38 @@ module.exports = handlePullRequestChange
 const isSemanticMessage = require('./is-semantic-message')
 const getConfig = require('probot-config')
 
-async function commitsAreSemantic (context) {
+async function commitsAreSemantic (context, allCommits = false) {
   const commits = await context.github.pullRequests.getCommits(context.repo({
     number: context.payload.pull_request.number
   }))
 
   return commits.data
-    .map(element => element.commit)
-    .some(commit => isSemanticMessage(commit.message))
+    .map(element => element.commit)[allCommits ? 'every' : 'some'](commit => isSemanticMessage(commit.message))
```

Comment:
This is operation to invoke either `every` or `some` on the array can be controversial. I'm happy to change it to a more verbose version if you like

Reply:
Clever!